Testscripts/Linux/TIME-CLOCKSOURCE.sh

```diff
@@ -28,15 +28,19 @@ CheckSource()
 	LogMsg "Running CheckSource"
 	current_clocksource="/sys/devices/system/clocksource/clocksource0/current_clocksource"
 
-	# Microsoft LIS installed version has lis_hyperv_clocksource_tsc_page or
-	#	lis_hv_clocksource_tsc_page.
-	# Without Microsoft LIS, hyperv_clocksource_tsc_page or hv_clocksource_tsc_page.
-	# CentOS 7.0 or older versions has hyperv_clocksource.
-	clocksource="v_clocksource_tsc_page"
-	source /etc/os-release
-	if [[ $DISTRO_NAME == "centos" || $DISTRO_NAME == "rhel" ]] && (($(echo "$VERSION_ID < 7.1" | bc -l))) || [[ $DISTRO == "redhat_6" || $DISTRO == "centos_6" ]]; then
-		clocksource="hyperv_clocksource"
+	# Microsoft LIS installed version for RH7.X has lis_hv_clocksource_tsc_page
```

Comment:
This cut-off information misleading the people spending time on the investigation every time. Can you add 'lis_hyperv_clocksource_tsc_page' here and below? Also, you add back 'CentOS 7.0 or older versions has hyperv_clocksource'

Reply:
I will add some for information to make it more clear.
I expect all distro (with LIS installed) between 7.0-7.4 to have ```lis_hyperv_clocksource_tsc_page``` If it not the case then test should fail as this is a modification to fix an issue.